Wake On Lan/Modules/HostList/Sources/HostListViewController.swift

```diff
@@ -54,9 +61,21 @@ public final class HostListViewController: UIViewController {
         navigationBar.prefersLargeTitles = true
     }
 
-    private func setupTableView() {
-        hostListView.tableView.delegate = presenter?.tableManager
-        hostListView.tableView.dataSource = presenter?.tableManager
+    private lazy var configureCellClosure = { (
```

Comment:
Definitely no. Such things as cell configurations and other UI features must be used only in view layer, but not in the view controller. So, let's move it back into the table manager.

Reply:
View controller it is smth like proxy. We shouldn't place whether UI configurations or other logic there. There are some exceptions of course (e.g. nav bar configurations or nav bar buttons), but no more.